client/src/sitemap.xml

```diff
@@ -780,6 +780,16 @@
     <changefreq>hourly</changefreq>
```

Comment:
I believe @jordanranz is fixing this, but can you nuke this file from the pull for now? It's just noise.